locales/en.js

```diff
@@ -75,5 +75,11 @@
 	"managerEmailNotExist": "Manager is not found, choose another one",
 	"mustBeManager": "You must be a manager to get access",
 	"pendingRequestsRetrieved": "All pending requests retrieved successfully",
-	"\"reason\" is not allowed": "\"reason\" is not allowed"
-}
```

Comment:
Eslint

Reply:
@RusimbiP The changes were removed but I fixed all ESLINT warnings